tests/security/cc/ipsec_client.pm

```diff
@@ -0,0 +1,68 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Run CC 'ipsec' client case
+# Maintainer: Liu Xiaojing <xiaojing.liu@suse.com>
+# Tags: poo#101226
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use audit_test;
+use Utils::Architectures;
+use lockapi;
+
+sub run {
+    my ($self) = @_;
+
+    select_console 'root-console';
+
+    my $server_ip = get_var('SERVER_IP', '10.0.2.101');
+    my $client_ip = get_var('CLIENT_IP', '10.0.2.102');
+
+    # We don't run setup_multimachine in s390x, but we need to know the server and client's
+    # ip address, so we add a known ip to eth0.
+    if (is_s390x) {
+        assert_script_run("ip addr add $client_ip/24 dev eth0");
```

Comment:
I would suggest not using hardcode 'eth0'

Reply:
Modified. thx